tests/test_config.py

```diff
@@ -10,6 +11,15 @@
 
 
 class ConfigTest(unittest.TestCase):
+    def _assertDatetimeAlmostEqual(self, s1, s2):
```

Comment:
Considering that both of these are strings, could we rename this method? Something like `_assertTimestampStringsAlmostEqual`. Or alternately, since the callers of this method do `strftime`, could we maybe make this method take `datetimes` directly?

Reply:
Excellent suggestion. I'll update the method name. 

If we wanted to keep the name the same and pass in datetime objects instead, we'd need to make more invasive changes to kinto/config/__init__.py or convert the stringified time back to datetime before passing to this method. I'll go with the less invasive approach to just change the method name and add a docstring for clarification.